### ISSUE TITLE
Copying families creates duplicates of instance variables and behaviors

### ISSUE THREAD

**Issue by dop2000** [Scirra/Construct-3-bugs#4707]
## Problem description

If you copy/paste a family with instance variables, the new family will have the same set of instance variables. 
Behaviors are also duplicated, even those that you can normally add only once, like Tween.

## Steps to reproduce

1. Copy and paste a family

## Observed result

![image](https://user-images.githubusercontent.com/36763350/113545636-e71c0500-962d-11eb-9951-f885a07855c1.png)
![image](https://user-images.githubusercontent.com/36763350/113545831-3d894380-962e-11eb-8439-e4904e883e0e.png)

## System details

<details><summary>View details</summary>
Platform information
Browser: Chrome
Browser version: 89.0.4389.114
Browser engine: Chromium
Browser architecture: 64-bit
Context: browser
Operating system: Windows
Operating system version: 10
Operating system architecture: 64-bit
Device type: desktop
Device pixel ratio: 1.25
Logical CPU cores: 4
Approx. device memory: 8 GB
User agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/89.0.4389.114 Safari/537.36
C3 release: r241 (stable)
Language setting: en-US

Local storage
Storage quota (approx): 127 gb
Storage usage (approx): 226 mb (0.2%)
Persistant storage: Yes

Browser support notes
This list contains missing features that are not required, but could improve performance or user experience if supported.

Nothing is missing. Everything is OK!
WebGL information
Version string: WebGL 2.0 (OpenGL ES 3.0 Chromium)
Numeric version: 2
Supports NPOT textures: yes
Supports GPU profiling: yes
Supports highp precision: yes
Vendor: Google Inc.
Renderer: ANGLE (Intel(R) HD Graphics 620 Direct3D11 vs_5_0 ps_5_0)
Major performance caveat: no
Maximum texture size: 16384
Point size range: 1 to 1024
Extensions:

EXT_color_buffer_float
EXT_color_buffer_half_float
EXT_disjoint_timer_query_webgl2
EXT_float_blend
EXT_texture_compression_bptc
EXT_texture_compression_rgtc
EXT_texture_filter_anisotropic
EXT_texture_norm16
KHR_parallel_shader_compile
OES_draw_buffers_indexed
OES_texture_float_linear
WEBGL_compressed_texture_s3tc
WEBGL_compressed_texture_s3tc_srgb
WEBGL_debug_renderer_info
WEBGL_debug_shaders
WEBGL_draw_instanced_base_vertex_base_instance
WEBGL_lose_context
WEBGL_multi_draw
WEBGL_multi_draw_instanced_base_vertex_base_instance
WEBGL_video_texture
WEBGL_webcodecs_video_frame
OVR_multiview2
Audio information
System sample rate: 48000 Hz
Output channels: 2
Output interpretation: speakers
Supported decode formats:

WebM Opus (audio/webm; codecs=opus)
Ogg Opus (audio/ogg; codecs=opus)
WebM Vorbis (audio/webm; codecs=vorbis)
Ogg Vorbis (audio/ogg; codecs=vorbis)
MPEG-4 AAC (audio/mp4; codecs=mp4a.40.5)
MP3 (audio/mpeg)
FLAC (audio/flac)
PCM WAV (audio/wav; codecs=1)
Supported encode formats:

WebM Opus (audio/webm; codecs=opus)
Video information
Supported decode formats:

WebM AV1 (video/webm; codecs=av01.0.00M.08)
MP4 AV1 (video/mp4; codecs=av01.0.00M.08)
WebM VP9 (video/webm; codecs=vp9)
WebM VP8 (video/webm; codecs=vp8)
Ogg Theora (video/ogg; codecs=theora)
H.264 (video/mp4; codecs=avc1.42E01E)
Supported encode formats:

WebM VP9 (video/webm; codecs=vp9)
WebM VP8 (video/webm; codecs=vp8)

</details>

**Comment by AshleyScirra** [comment]
Minimal repro: [issue4707.zip](https://github.com/Scirra/Construct-3-bugs/files/6263813/issue4707.zip)
Steps to reproduce:

1. Right click 'Family1' in project bar and copy
2. Right click 'Families' folder and paste (note it appears to work correctly if you right click a family and paste - issue appears to only happen if pasting on root Families folder)

Expected result: error pasting as it will cause a name collision with instance variables (as seems to be correct pasting on a family)
Observed result: incorrectly allows paste to go ahead

(please note it saves time if you can provide this info as per the bug report guidelines)

**Comment by AshleyScirra** [comment]
OK, this was quite tricky but should be fixed in the next release. The fix just prevents the paste as it would create a name collision. This does block copy/paste of some families in the same project, but you can still copy/paste them to a different project.
